zfit/core/constraint.py

```diff
@@ -184,10 +183,11 @@ def __init__(self, params: ztyping.ParamTypeInput, mu: ztyping.NumericalScalarTy
 
         self._covariance = covariance
         self._mu = mu
+        self._params_array = params
 
         distribution = tfd.MultivariateNormalFullCovariance
         dist_params = dict(loc=mu, covariance_matrix=covariance)
-        dist_kwargs = dict(validate_args=True)
+        dist_kwargs = dict(validate_args=False)
```

Comment:
Why remove this check except for possible improvement in runtime performance?

Reply:
Good catch, thx! Forgot to change that back (done now)